<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_material</class>
 <widget class="QMainWindow" name="edit_material">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>680</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>680</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Επεξεργασία Υλικού</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>media/gear.png</normaloff>media/gear.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="update_material_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>535</y>
      <width>150</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Επεξεργασία Υλικού</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_material_fields">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>535</y>
      <width>150</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Καθαρισμός </string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Calligraphy</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Συμπληρώστε τα στοίχεια για επεξεργασία υλικού.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_material">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2_material">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>170</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3_material">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>220</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4_material">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>270</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5_material">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>320</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_material_xaraktiristika">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>350</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Βασικά Χαρακτηριστικά :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_material_xaraktiristika_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>350</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ποσότητα Ομοίων Αντικειμένων :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_material_xaraktiristika_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>350</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Κατάσταση :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_material_xaraktiristika_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>350</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Εταιρία :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_material_xaraktiristika_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>350</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Μοντέλο :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_material_xaraktiristika_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>350</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Λέξεις Κλειδιά(Χωρίστε με κόμμα):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6_material">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>370</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_material_xaraktiristika_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>350</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Κατηγορία:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_material_xaraktiristika_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>350</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Υποκατηγορία:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7_material">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>420</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_8_material">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>470</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
